apps/web/modules/bookings/components/BookingsCalendar.tsx

```diff
@@ -26,9 +79,7 @@ export function BookingsCalendar({ table }: BookingsCalendarViewProps) {
           <DataTableSegment.Select />
         </div>
       </div>
-      <div className="flex items-center justify-center pt-2 xl:pt-0">
-        <EmptyScreen Icon="calendar" headline="Calendar view" description="Calendar view is coming soon." />
-      </div>
+      <BookingsCalendarView bookings={bookings} onWeekChange={handleWeekChange} />
```

Comment:
While `BookingsCalendar` takes care of preparing `bookings` array and extending `dateRange` filter value, `BookingsCalendarView` is more presentational with little business logic.